src/RefractAPI.cc

```diff
@@ -153,7 +153,7 @@ namespace drafter {
 
         // Add default value
         if (!parameter.node->defaultValue.empty()) {
-            element->attributes[SerializeKey::Default] = CreateArrayElement(LiteralToRefract<T>(MAKE_NODE_INFO(parameter, defaultValue), context), true);
+            element->attributes[SerializeKey::Default] = LiteralToRefract<T>(MAKE_NODE_INFO(parameter, defaultValue), context);
```

Comment:
Since this is run only for `enum`, default should always be an array. Thus, this change is wrong